plugins/scaffolder/src/legacy/TemplateEditorPage/TemplateEditorForm.tsx

```diff
@@ -110,23 +107,14 @@ export function TemplateEditorForm(props: TemplateEditorFormProps) {
 
   const [steps, setSteps] = useState<TemplateParameterSchema['steps']>();
 
-  const fields = useMemo(() => {
-    return Object.fromEntries(
-      fieldExtensions.map(({ name, component }) => [name, component]),
-    );
-  }, [fieldExtensions]);
-
   useDebounce(
     () => {
       try {
         if (!content) {
           setSteps(undefined);
           return;
         }
-        const parsed: JsonValue = yaml
-          .parseAllDocuments(content)
-          .filter(c => c)
-          .map(c => c.toJSON())[0];
+        const parsed: JsonValue = yaml.parse(content);
```

Comment:
why removed this ? was to fix this #18778

Reply:
oh nice catch, think I ended up copying the wrong files here to replace these ones will fix this up tomorrow.